[2024-04-02T21:14:10.205+0800] {processor.py:161} INFO - Started process (PID=19296) to work on /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:14:10.206+0800] {processor.py:830} INFO - Processing file /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-02T21:14:10.206+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:14:10.206+0800] {dagbag.py:540} INFO - Filling up the DagBag from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:14:10.221+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:14:10.220+0800] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-02T21:14:10.221+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:14:10.221+0800] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-02T21:14:10.221+0800] {processor.py:842} WARNING - No viable dags retrieved from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:14:10.249+0800] {processor.py:183} INFO - Processing /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2024-04-02T21:15:47.435+0800] {processor.py:161} INFO - Started process (PID=19753) to work on /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:15:47.436+0800] {processor.py:830} INFO - Processing file /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-02T21:15:47.436+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:15:47.436+0800] {dagbag.py:540} INFO - Filling up the DagBag from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:15:47.451+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:15:47.451+0800] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-02T21:15:47.451+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:15:47.451+0800] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-02T21:15:47.451+0800] {processor.py:842} WARNING - No viable dags retrieved from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:15:47.478+0800] {processor.py:183} INFO - Processing /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2024-04-02T21:19:24.521+0800] {processor.py:161} INFO - Started process (PID=20372) to work on /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:19:24.521+0800] {processor.py:830} INFO - Processing file /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-02T21:19:24.522+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:19:24.522+0800] {dagbag.py:540} INFO - Filling up the DagBag from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:19:24.534+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:19:24.534+0800] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-02T21:19:24.534+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:19:24.534+0800] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-02T21:19:24.535+0800] {processor.py:842} WARNING - No viable dags retrieved from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:19:24.560+0800] {processor.py:183} INFO - Processing /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-04-02T21:21:00.223+0800] {processor.py:161} INFO - Started process (PID=20485) to work on /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:21:00.223+0800] {processor.py:830} INFO - Processing file /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-02T21:21:00.224+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:21:00.223+0800] {dagbag.py:540} INFO - Filling up the DagBag from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:21:00.237+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:21:00.236+0800] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-02T21:21:00.237+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:21:00.237+0800] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-02T21:21:00.237+0800] {processor.py:842} WARNING - No viable dags retrieved from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:21:00.262+0800] {processor.py:183} INFO - Processing /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-04-02T21:31:11.946+0800] {processor.py:161} INFO - Started process (PID=22230) to work on /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:31:11.946+0800] {processor.py:830} INFO - Processing file /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-02T21:31:11.947+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:31:11.947+0800] {dagbag.py:540} INFO - Filling up the DagBag from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:31:11.962+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:31:11.962+0800] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-02T21:31:11.962+0800] {logging_mixin.py:188} INFO - [2024-04-02T21:31:11.962+0800] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-02T21:31:11.962+0800] {processor.py:842} WARNING - No viable dags retrieved from /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-02T21:31:11.991+0800] {processor.py:183} INFO - Processing /Users/kalerbramastha/Library/Mobile Documents/com~apple~CloudDocs/bram/career_assessment/akasia_de/python-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
